CoreWelcomeView/src/au/gov/asd/tac/constellation/views/welcome/WelcomeViewPane.java

```diff
@@ -198,14 +209,55 @@ public WelcomeViewPane() {
                 }
                 final String text = recentGraphButtons[i].getText();
 
-                final Rectangle2D value = new Rectangle2D(700, 150, 500, 500);
+                final Rectangle2D value = new Rectangle2D(0, 0, 145, 145);
                 final String screenshotFilename = RecentGraphScreenshotUtilities.getScreenshotsDir() + File.separator + text + ".png";
+                final String screenshotFilenameResize = RecentGraphScreenshotUtilities.getScreenshotsDir() + File.separator + text + "Resize" + ".png";
+
                 if (new File(screenshotFilename).exists()) {
-                    final ImageView imageView = new ImageView(new Image("File:/" + screenshotFilename));
+
+                    final int IMAGE_SIZE = 145;
+
+                    if (!new File(screenshotFilenameResize).exists()) {
+                        Path source = Paths.get(screenshotFilename);
+                        Path target = Paths.get(screenshotFilenameResize);
+
+                        try (InputStream is = new FileInputStream(source.toFile())) {
+                            BufferedImage originalImage = ImageIO.read(is);
+
+                            // create a new BufferedImage for drawing
+                            BufferedImage newResizedImage
+                                    = new BufferedImage(IMAGE_SIZE, IMAGE_SIZE, BufferedImage.TYPE_INT_ARGB);
+                            Graphics2D g = newResizedImage.createGraphics();
+
+                            g.setComposite(AlphaComposite.Src);
+                            g.fillRect(0, 0, IMAGE_SIZE, IMAGE_SIZE);
+
+                            Map<RenderingHints.Key, Object> hints = new HashMap<>();
+                            hints.put(RenderingHints.KEY_RENDERING, RenderingHints.VALUE_RENDER_QUALITY);
+                            hints.put(RenderingHints.KEY_INTERPOLATION, RenderingHints.VALUE_INTERPOLATION_BILINEAR);
+                            hints.put(RenderingHints.KEY_ANTIALIASING, RenderingHints.VALUE_ANTIALIAS_ON);
+                            g.addRenderingHints(hints);
+
+                            // puts the original image into the newResizedImage
+                            g.drawImage(originalImage, 0, 0, IMAGE_SIZE, IMAGE_SIZE, null);
+                            g.dispose();
+
+                            // get file extension
+                            String s = target.getFileName().toString();
+                            String fileExtension = s.substring(s.lastIndexOf(".") + 1);
+
+                            // we want image in png format
+                            ImageIO.write(newResizedImage, fileExtension, target.toFile());
+                        } catch (Exception e) {
+
+                        }
```

Comment:
Good that you mentioned you got this from @mkyong, he is awesome. Please put this code block into a method and in the javadoc for the method reference the url (https://mkyong.com/java/how-to-resize-an-image-in-java/) so he gets credit. 

We use `Graphics2D` elsewhere in the code and I know it used in `GlyphManagerBI`'s `buildLigature()` so that is fine.

Add some `final`'s to variable you can in here or else @antares1470 will get you!